examples/preprocessing/plot_power_transformer.py

```diff
@@ -0,0 +1,85 @@
+"""
+==========================================================
+Using PowerTransformer to apply the Box-Cox transformation
+==========================================================
+
+This example demonstrates the use of the Box-Cox transform
+through ``PowerTransformer`` to map data from various
+distributions to a Gaussian distribution.
+
+Box-Cox is useful as a transformation in modeling problems
+where homoscedasticity and normality are desired. Below
+are examples of Box-Cox applied to three distributions: a
+heavily left-skewed lognormal distribution, a left-skewed
+chi-squared distribution, and a right-skewed Weibull
```

Comment:
Chi

Reply:
chi-squared should be fine: https://en.wikipedia.org/wiki/Chi-squared_distribution